client/src/pages/Dashboard.js

```diff
@@ -5,31 +5,32 @@ import { getUserInfo } from '../redux/actions/userActions';
 import { getTwitterProfileInfoData } from '../redux/actions/twitterActions';
 import DisplayUserMessages from '../components/DisplayUserMessages';
 import ConnectTwitterButton from '../components/ConnectTwitterButton';
+import TwitterTimeline from '../components/TwitterTimeline';
 
 const Dashboard = () => {
   useSelector((state) => state.userReducer.present);
-  const isConnected = useSelector((state) => state.twitterReducer.isConnected);
+  const { isConnected } = useSelector((state) => state.twitterReducer);
   const dispatch = useDispatch();
 
   useEffect(() => {
     dispatch(getUserInfo());
   }, [dispatch]);
 
   return (
-    <div>
```

Comment:
Very good choosing not to add unnecessary nodes in your dom.
*NOT CHANGES REQUIRED*